rainbow-me/rainbow
# Profiles: `cover` to `header`

Fixes TEAM2-382 TEAM2-355

## What changed (plus any additional context for devs)

This PR changes usages of the `cover` text record key to `header`.

Note: previously set cover photos will be cleared due to the key change.

## Screen recordings / screenshots

N/A

## What to test

- Ensure that cover photos still upload on the create/edit profile screen
- Ensure that cover photos are applied via a `header` text record on the ENS when submitted
- Ensure that cover photos are still viewable on the View Profile screen
